### ISSUE TITLE
clear input field after making a move

### ISSUE THREAD

**Issue by Jan-de-Munck** [everyonesdesign/Chess-Helper#67]
basically the title, so you don't have  to press backspace 2-4 times after every move

**Comment by Jan-de-Munck** [comment]
it should just be adding  `input.value = "'';` on (new) line 87 of Chess-Helper/app/src/keyboard.ts but I'm not too familiar with git hub so I don't know if it's possible for me to make a merge request.

**Comment by everyonesdesign** [comment]
Hello, @Jan-de-Munck !
Could you check in which version you have the issue? A similar issue was fixed in version 6.3.1

**Comment by Jan-de-Munck** [comment]
ah, I just installed it of the firefox extension manager and it seems to be fixed now. Seems there was an update yesterday witch must've solved it

**Comment by Jan-de-Munck** [comment]
edit: nvm, it worked on daily puzzle, but in an online match it doesn't. when I press enter, the move gets done but the imput field doesn't clear. So  i have to either press esc or backspace a few times. Firefox says version is 5.5.2 and last updated 2023-04-10 (yesterday)

**Comment by everyonesdesign** [comment]
Yes, you're right, the new version hasn't been released yet for Firefox.

5.5.2 is a very old version (May 9, 2021). Mozilla staff is reviewing the new version of the extension:

![image](https://user-images.githubusercontent.com/3520704/231153370-08e5032b-1934-4dc0-a79e-b0f75ca3657a.png)

I don't know how long the review will take on their side. Feel free to keep the issue open until it's reviewed by Mozilla and then manually tested.

**Comment by everyonesdesign** [comment]
Hello! Version 6.3.2. went live in Mozilla store. Could you, please, check the issue? Thanks!

For now I'll mark the issue as solved, but feel free to reopen if reproduced again with the new version.
